@xen-orchestra/fs/src/local.js

```diff
@@ -92,7 +96,10 @@ export default class LocalHandler extends RemoteHandlerAbstract {
   }
 
   async _readFile(file, options) {
-    return fs.readFile(this._getFilePath(file), options)
+    return retry(() => fs.readFile(this._getFilePath(file), options), {
```

Comment:
It's probably unnecessary to retry `this._getFilePath(file)`?

Reply:
Since we added `retry` on `fs.readFile`, the `EAGAIN` error on `open /...` has not reappeared for the moment